src/app/shared/card/card-prescriptions/card-prescriptions.component.html

```diff
@@ -0,0 +1,20 @@
+<div *ngIf="data" class="card-treatment">
+  <div class="title-container">
+    <div class="title">{{ drugInfo[0] }}</div>
```

Comment:
is this correct? drugInfo[0], drugInfo[1] ... what if drugInfo is empty array? an error will appear

Reply:
drugInfo cannot be an empty array, as some of the field values which are concatenated in it are required fields. Anyway, tested it with [] and no error appeared.